shared/manifest.go

```diff
@@ -0,0 +1,57 @@
+// Copyright 2018 The WPT Dashboard Project. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package shared
+
+import "strings"
+
+const (
+	windowJSSuffix                 = ".window.js"
+	windowHTMLSuffix               = ".window.html"
+	anyJSSuffix                    = ".any.js"
+	anyJSWindowHTMLSuffix          = ".any.html"
+	anyJSDedicatedWorkerHTMLSuffix = ".any.worker.html"
+	anyJSServiceWorkerHTMLSuffix   = ".any.serviceworker.html"
+	anyJSSharedWorkerHTMLSuffix    = ".any.sharedworker.html"
```

Comment:
There are in fact two types of different constants here: `.window.js` and `.any.js` are the keys, while the rest `*.html` are the expanded values, so perhaps we should define a map instead.

Besides, you are missing `.worker.js`.

Reply:
Changed to a map + added worker, PTAL